Mudz.Common/Domain/Environment/Map/Grid.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Mudz.Common.Domain.Environment.Map.Room;
```

Comment:
Not sure why some files showing as an Add in the diff. Respective *removes* can be found in overall diff. Github should just show case change to namespace so blaming github on this one.